integration_test/agents/agents.go

```diff
@@ -664,9 +693,16 @@ func InstallStandaloneWindowsMonitoringAgent(ctx context.Context, logger *log.Lo
 }
 
 func getRestartOpsAgentCmd(imageSpec string) string {
-	if gce.IsWindows(imageSpec) {
+	isUAPPlugin := IsOpsAgentUAPPlugin()
```

Comment:
I think the logic it will be easier to understand if we separated uap from the rest
if isUAPplugin {
    if windows { return "" }
    return grpcul ...
}